Tests.Integration.Presentation.Web/Contract/ItContractTest.cs

```diff
@@ -185,5 +185,19 @@ public async Task Cannot_Create_With_Duplicate_Name_In_Same_Organization()
             //Assert
             Assert.Equal(HttpStatusCode.Conflict, conflictedResponse.StatusCode);
         }
+
+        [Fact]
+        public async Task Can_Get_Organization_Filtered_By_Name()
```

Comment:
this test says it tests Get_Organization_By_Name but it seems to check for unique naming...